helix-term/src/ui/menu.rs

```diff
@@ -320,10 +320,8 @@ impl<T: Item + 'static> Component for Menu<T> {
             let is_marked = i >= scroll_line && i < scroll_line + scroll_height;
```

Comment:
@archseer Reading the code makes me think that this is actually supposed to be a multiline scroll bar but there's a bug, is that so?

Reply:
It is multiline for me:

![hx-completion-menu](https://user-images.githubusercontent.com/96927121/173201617-1db36487-6b8e-46b4-a87f-34ba9c99776d.png)

`git describe` -> `22.05-49-g567e71fb`

Reply:
Yeah it's multi-line depending on total list length/page size

Reply:
Ah what confused me was that a shorter scrollbar usually means longer content and a longer scrollbar means shorter content in other applications, but here it seems to be inverted. I think inverting it will be more intuitive.

Reply:
Fixed in #2860, twas a typo in the floor division impl :)